packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -158,6 +160,8 @@ export function NetworksAndSignersProvider(
     status: defaultStatus
   })
   const [tosAccepted] = useLocalStorage<string>(TOS_LOCALSTORAGE_KEY)
+  const isConnectedToArbitrum = useIsConnectedToArbitrum()
```

Comment:
Might be too much for this PR, but could we replace most of this logic with `useProvider/useSigner`?

Reply:
Yeah but let's not do it now